CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+Add: Support new commands payloadType - plain text 
```

Comment:
```suggestion
- Add: Support new commands payloadType - plain text (#716)
```

Reply:
> 

I do not know if your suggestion includes a typo. Just to clarify, a new payloadType (`text`, used to return plain text) was added to codebase. Additionally, other tests were included, and doc extended. 

So, fixing the typo your message, it could be:

```suggestion
- Add: support new commands payloadType "text" (for plain text) (#716)
```

Reply:
Yes. It was a typo :) Commited suggestion ([cd2402c](https://github.com/telefonicaid/iotagent-json/pull/720/commits/cd2402c228279c7ef3d296a77e5f740ee78fca62))